File: root-config/ios/App/App/EchoPlugin.m
//
//  EchoPlugin.m
//  App
//
//  Created by Elliott Davies on 16/03/2023.
//

#import <Capacitor/Capacitor.h>

CAP_PLUGIN(EchoPlugin, "Echo",
    CAP_PLUGIN_METHOD(echo, CAPPluginReturnPromise);
)

File: root-config/ios/App/App/EchoPlugin.swift
//
//  EchoPlugin.swift
//  App
//
//  Created by Elliott Davies on 16/03/2023.
//

import Capacitor

@objc(EchoPlugin)
public class EchoPlugin: CAPPlugin {
    @objc func echo(_ call: CAPPluginCall) {
        let value = call.getString("value") ?? ""
        call.resolve(["value": value])
    }
}
